### DISCUSSION TITLE
ConnectionError: Could not connect to `mongod`

### DISCUSSION THREAD

**Discussion by byungjinku** [obss/sahi#941] [Q&A]
---------------------------------------------------------------------------
ServerSelectionTimeoutError               Traceback (most recent call last)
[/usr/local/lib/python3.10/dist-packages/fiftyone/core/odm/database.py](https://localhost:8080/#) in _validate_db_version(config, client)
    289     try:
--> 290         version = Version(client.server_info()["version"])
    291     except Exception as e:

14 frames
ServerSelectionTimeoutError: localhost:27017: [Errno 111] Connection refused, Timeout: 30s, Topology Description: <TopologyDescription id: 64f1716991a0b2462ef571e8, topology_type: Unknown, servers: [<ServerDescription ('localhost', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('localhost:27017: [Errno 111] Connection refused')>]>

The above exception was the direct cause of the following exception:

ConnectionError                           Traceback (most recent call last)
[/usr/local/lib/python3.10/dist-packages/fiftyone/core/odm/database.py](https://localhost:8080/#) in _validate_db_version(config, client)
    291     except Exception as e:
    292         if isinstance(e, ServerSelectionTimeoutError):
--> 293             raise ConnectionError("Could not connect to `mongod`") from e
    294 
    295         raise RuntimeError("Failed to validate `mongod` version") from e

ConnectionError: Could not connect to `mongod`

I couldn't resolve this problem

**Comment by jokober** [comment]
I suppose this happens when you run fiftyone? What OS are you using?

**Reply by byungjinku** [reply]
Is Window
